Dockerfile

```diff
@@ -1,23 +1,71 @@
-# AUTHOR: Roberto Vitillo
-# DESCRIPTION: Mozilla's Airflow container
-# BUILD: docker build --rm -t mozdata/telemetry-airflow
-# SOURCE: https://github.com/mozilla/telemetry-airflow
-
-FROM puckel/docker-airflow:1.7.1.3
-MAINTAINER vitillo
-
-USER root
-RUN apt-get update -yqq && \
-    apt-get install -yqq python-pip python-mysqldb && \
-    pip install boto3 && \
-    pip install airflow[async] && \
-    pip install airflow[password] && \
-    pip install retrying
-
-ADD ansible/files/airflow/airflow.cfg ${AIRFLOW_HOME}/airflow.cfg
-ADD ansible/files/airflow/entrypoint.sh ${AIRFLOW_HOME}/entrypoint.sh
-ADD ansible/files/airflow/replace_env.py ${AIRFLOW_HOME}/replace_env.py
-RUN chown airflow:airflow ${AIRFLOW_HOME}/airflow.cfg
-
-USER airflow
-ADD dags/ /usr/local/airflow/dags/
+FROM python:2-slim
+MAINTAINER Jannis Leidel <jezdez@mozilla.com>
+
+ENV PYTHONUNBUFFERED=1 \
+    AWS_REGION=us-west-2 \
+    # AWS_ACCESS_KEY_ID= \
+    # AWS_SECRET_ACCESS_KEY= \
+    SPARK_BUCKET=telemetry-spark-emr-2 \
+    AIRFLOW_BUCKET=telemetry-test-bucket \
+    PRIVATE_OUTPUT_BUCKET=telemetry-test-bucket \
+    PUBLIC_OUTPUT_BUCKET=telemetry-test-bucket \
+    EMR_KEY_NAME=mozilla_vitillo \
+    EMR_FLOW_ROLE=telemetry-spark-cloudformation-TelemetrySparkInstanceProfile-1SATUBVEXG7E3 \
+    EMR_SERVICE_ROLE=EMR_DefaultRole \
+    EMR_INSTANCE_TYPE=c3.4xlarge \
+    PORT=8000
+
+ENV AIRFLOW_HOME=/app \
+    AIRFLOW_AUTHENTICATE=True \
+    AIRFLOW_AUTH_BACKEND=airflow.contrib.auth.backends.google_auth \
+    AIRFLOW_BROKER_URL=redis://redis:6379/0 \
+    AIRFLOW_RESULT_URL=${AIRFLOW_BROKER_URL} \
+    AIRFLOW_FLOWER_PORT="5555" \
+    AIRFLOW_DATABASE_URL=postgres://postgres@db/postgres \
+    AIRFLOW_FERNET_KEY="VDRN7HAYDw36BTFbLPibEgJ7q2Dzn-dVGwtbu8iKwUg" \
+    AIRFLOW_SECRET_KEY="3Pxs_qg7J6TvRuAIIpu4E2EK_8sHlOYsxZbB-o82mcg" \
```

Comment:
@jezdez it's not obvious that this isn't an actual secret. Should we change it to something more obviously fake?